apps/dashboard/src/pages/integrations/components/integration-card.tsx

```diff
@@ -0,0 +1,117 @@
+    <div
+      className={cn(
+        'bg-card shadow-xs group relative flex min-h-[125px] cursor-pointer flex-col gap-2 overflow-hidden rounded-xl border border-neutral-100 p-3 transition-all hover:shadow-lg',
+        !integration.active && 'opacity-75 grayscale'
+      )}
+      onClick={handleConfigureClick}
+      data-test-id={`integration-${integration._id}-row`}
+    >
+      <div className="flex items-start justify-between">
+        <div className="flex items-center gap-1.5">
+          <div className="relative h-6 w-6">
+            <ProviderIcon
+              providerId={provider.id}
+              providerDisplayName={provider.displayName}
+              className="h-full w-full"
+            />
+          </div>
+          <span className="text-sm font-medium">{integration.name}</span>
+        </div>
+        {integration.primary && (
+          <Tooltip>
+            <TooltipTrigger>
+              <RiStarSmileLine className="text-feature h-4 w-4" />
+            </TooltipTrigger>
+            <TooltipContent>This is your primary integration for the {provider.channel} channel.</TooltipContent>
+          </Tooltip>
+        )}
+      </div>
+      <div className="flex items-center gap-2">
+        {isDemo && (
+          <Tooltip>
+            <TooltipTrigger className="flex h-[16px] items-center gap-1">
+              <span className="flex h-[16px] items-center gap-1">
+                <Badge variant="warning" className="text-2xs h-[16px] rounded-sm text-[#F6B51E]">
+                  DEMO
+                </Badge>
+              </span>
+            </TooltipTrigger>
+            <TooltipContent>
+              <p>
+                This is a demo provider for testing purposes only and capped at 300{' '}
+                {provider.channel === 'email' ? 'emails' : 'sms'} per month. Not suitable for production use.
+              </p>
+            </TooltipContent>
+          </Tooltip>
+        )}
+      </div>
+
+      <div className="mt-auto flex items-center gap-2">
+        {integration.channel === 'in_app' && !integration.connected ? (
+          <Button size="xs" className="h-[26px]" variant="outline" onClick={handleConfigureClick}>
+            <RiSettings4Line className="h-4 w-4" />
+            Configure
+          </Button>
+        ) : (
+          <Badge variant={integration.active ? 'success' : 'neutral'} className="capitalize">
+            <RiCheckboxCircleFill className="text-success h-4 w-4" />
+            {integration.active ? 'Active' : 'Inactive'}
+          </Badge>
+        )}
+        <Badge variant="outline" className="shadow-none">
+          <RiGitBranchFill
+            className={cn('h-4 w-4', environment.name.toLowerCase() === 'production' ? 'text-feature' : 'text-warning')}
+          />
+          {environment.name}
+        </Badge>
+      </div>
+    </div>
+  );
+}
+
+export function isDemoIntegration(providerId: string) {
+  return providerId === 'novu-email' || providerId === 'novu-sms';
+}
```

Comment:
no need for the export, and please define it with using enums and move it to the utils